src/form-builder/types.ts

```diff
@@ -71,3 +71,16 @@ export interface OMBInfoProps {
  *
  */
 export type FullNameProps = FieldProps<string>;
+
+export type AddressProps = FieldProps<string>
```

Comment:
I think this was discussed at some point but why not use an interface here as well?